src/components/m-header-sub-navigation/_template.js

```diff
@@ -3,7 +3,7 @@ import raw from 'bel/raw';
 
 const linkItem = ({ url, name, isActive }) => bel`
   <li class="m-header-sub-navigation__list-item">
-    <a class="m-header-sub-navigation__link js-header-navigation-close ${isActive ? 'is-header-sub-navigation-active' : ''}" href="${url}">${raw(name)}</a>
+    <a class="m-header-sub-navigation__link ${isActive ? 'is-header-sub-navigation-active' : ''}" href="${url}">${raw(name)}</a>
```

Comment:
Will may also need migration for axa.ch

Reply:
axa ch does not use that class

Reply:
<img width="850" alt="bildschirmfoto 2018-03-07 um 17 14 04" src="https://user-images.githubusercontent.com/275235/37103635-f7c989f6-222a-11e8-93a3-78eebacaecc5.png">